examples/medplum-chart-demo/src/components/Allergies.tsx

```diff
@@ -62,7 +67,12 @@ export function Allergies(props: AllergiesProps): JSX.Element {
       <Modal opened={opened} onClose={close} title="Add Allergy">
         <Form onSubmit={handleSubmit}>
           <Stack>
-            <TextInput name="allergy" label="Allergy" data-autofocus={true} required autoFocus />
+            <CodingInput
```

Comment:
I recommend using `<CodeableConceptInput>` here rather than `<CodingInput>`.  Currently you're using `<CodingInput>` and manually creating a `CodeableConcept` -- might as well use the tool to do the work.